### ISSUE TITLE
Does this work on logits? How about the dimensions of the output and target?

### ISSUE THREAD

**Issue by sarmientoj24** [oikosohn/compound-loss-pytorch#4]
Can it work if output is in B x C x H x W and target is B x H x W?
